init:
  vehicle_number: 1
  task_number: 10
  linear_scale: 0.5
  angular_scale: 0.5
